my-notes.md

```diff
@@ -0,0 +1,123 @@
+# My notes
+
+## Considered packages
+
+* [elm-form](https://package.elm-lang.org/packages/dillonkearns/elm-form/3.0.1/) by `dillonkearns`
+  * Pros
+    * Every possible field kind is supported
+    * Supports several forms on one page
+    * "Monadic" way of validation (with `andMap` / `andThen`)
+  * Cons
+    * In Ellie example, forms doesn't show required fields from the start, I for sure can do it myself though, may be it is the way the example was written and not the limitation
+    * It seems that rendering can not be customized
+    * Form functions get their parameters as arguments, for a lot of fields in the form there could be too much
+      * ...but may be it is possible to use records
+* [elm-form](https://package.elm-lang.org/packages/etaque/elm-form/4.0.0/) by `etaque`
+  * Pros
+    * I like the API as well, even more than API above
+    * Also supports all the fields
+    * Includes API for tests
+  * Cons
+    * Seems outdated and abandoned
+    * Also doesn't show required fields at start
+    * Not completely typed as uses strings as field names
+    * Has some issues with TextArea, though it seems we don't need textareas here
+* [elm-validate](https://package.elm-lang.org/packages/rtfeldman/elm-validate/4.0.2/) by _Richard Feldman_
+  * Pros
+    * The _right_ way, as from Feldman
```

Comment:
Watch out. Feldman has a lot of passion and pull on other people, but his designs can be improved on.

Reply:
> newtype

I *thought* you smelled of Haskell... XD

Reply:
I've been made aware that my comment above might come off as patronizing.
That's what I wanted, but I see that it could be legitimately read that way.
My apologies. >_<